### PR TITLE
Separate frontend and backend service and rm datastore package

### LINKED ISSUE #109: Separate frontend and backend service.
Currently we serve UI as static files in Go server. However it has routing problem as discussed in https://github.com/oam-dev/velacp/issues/107.

We should use nodejs to run UI code and provide better support on local dev. Also need to modify contributing doc: https://github.com/oam-dev/velacp/blob/main/docs/contributing/backend.md and https://github.com/oam-dev/velacp/blob/main/docs/contributing/frontend.md

### PR DESCRIPTION
**Pull request by wangyuan249**
1. fix #109 
2. remove datastore package

### CODE REVIEW

File: ui/config/proxy.ts [hunk 1]
```diff
@@ -5,24 +5,29 @@
  * For details, please see
  * https://pro.ant.design/docs/deploy
  */
+const { PROXY_PORT } = process.env
```
Comment (hongchaodeng) [inline ui/config/proxy.ts:8]: set a default port?

File: ui/config/proxy.ts [hunk 2]
```diff
@@ -5,24 +5,29 @@
...
+const PRE_TAEGET = `http://localhost:${PROXY_PORT}`
+
 export default {
   dev: {
-    '/api/': {
-      target: 'https://preview.pro.ant.design',
+    '/api': {
+      target: DEV_TAEGET,
       changeOrigin: true,
       pathRewrite: { '^': '' },
```
Comment (hongchaodeng) [inline ui/config/proxy.ts:18]: See this doc: https://umijs.org/config#proxy

File: ui/config/proxy.ts (hunk 1 above)
Comment (wangyuan249) [reply ui/config/proxy.ts]: ok  have changed to `process.env.PROXY_PORT || 8000`

File: ui/config/proxy.ts (hunk 2 above)
Comment (wangyuan249) [reply ui/config/proxy.ts]: have fixed

**Review by hongchaodeng — APPROVED** [review]
lgtm

**Pull request merged**